meta-ostro/classes/supported-recipes.bbclass

```diff
@@ -270,8 +364,14 @@ python supported_recipes_eventhandler() {
             collections = supported_recipes.find_collections(pn)
             return ' (would be supported in %s)' % ' '.join(collections) if collections else ''
 
+        def re_escape(str):
+            # In contrast to re.escape(), we only escape + (as in gtk+3).
+            # Escaping all non-alphanumerics makes many entries (like linux-yocto)
+            # unnecessarily less readable (linux\-yocto).
+            return ''.join(['\\+' if x == '+' else x for x in str])
+
```

Comment:
such small repetition is not that problematic. I think other function with only line of "beef" is usually not justified. In the end of the day, it's python, not java for all those bloated getters and setters :)